File: 16.08.2021/python/task2.py
import math


print('a, b, c енгіз (пробел арқылы)')

arr = list(map(int, input().split())) 

p = (arr[0] + arr[1] + arr[2]) / 2

print(f'p = {p}')
s = math.sqrt(p * (p - arr[0]) * (p - arr[1]) * (p - arr[2]))

print(f's = {s}')

File: 16.08.2021/cpp/task2.cpp
#include <iostream>
# include "math.h"


using namespace std;

int main() {
	short int a, b, c;
	float p;
	
	cout << "a, b, c: "; cin >> a >> b >> c;
	p = (a + b + c) / 2;

	cout <<"s = " << sqrt(p * (p - a) * (p - b) * (p - c));
	
	return 0;
}

